### DISCUSSION TITLE
Email

### DISCUSSION THREAD

**Discussion by Kushwaha2021** [saran-surya/email_auth_node#20] [Q&A]
Why there is no "to: xyz@gmail.com" display when user receive email ?

**Comment by saran-surya** [comment]
Hi @Kushwaha2021 Can you please eloborate you query ?

**Comment by Kushwaha2021** [comment]
I want to display recipient mail but as of now it's only display "
***@***.***" not ***@***.***".
So how can I do that.

**Reply by saran-surya** [reply]
So where do you want to display it, can you explain it completely, small bits of information will not help, kindly elaborate it properly, where do you want to show the mail id, and where is it displaying, atleast add some images to explain.

Thanks

**Comment by Kushwaha2021** [comment]
please find attachment. here "to:" field is empty but it should be filled
with ***@***.*** address.

**Reply by saran-surya** [reply]
Sorry I do not see any images, can you please repeat the same from GitHub ?

**Comment by Kushwaha2021** [comment]
sure. PFA

**Comment by Kushwaha2021** [comment]
"to:" field is empty but it should be filled with ***@***.***
***@***.***> address.

**Reply by saran-surya** [reply]
### Hi, I still don't see any images in GitHub, Can you please do it again. And wait for the image to upload before posting it.

**Reply by Kushwaha2021** [reply]
> ### Hi, I still don't see any images in GitHub, Can you please do it again. And wait for the image to upload before posting it.

![Screenshot (143)_LI](https://user-images.githubusercontent.com/91123190/147675159-1fecdad9-ebb6-47f8-a942-8e32c795043b.jpg)

**Reply by Kushwaha2021** [reply]
> 

"to:" this field is empty it should have recipient mail address.

**Reply by saran-surya** [reply]
Okay got it, So the sender is by default set to bcc, So the name will not be visible, If so you want them to be visible, The email_auth_node package can be changed, As per a issue, If that is okay Kindly create a feature request, and it will be taken care of.

Thanks

**Reply by Kushwaha2021** [reply]
Do I have to pull request or anything for that?

**Reply by saran-surya** [reply]
It will be appreciated  if you can contribute, with an option to set the sender to be visible, Else you can create a new feature issue, And it will be taken care of.

**Comment by Kushwaha2021** [comment]
Ok, thanks.
